package.json

```diff
@@ -80,10 +82,12 @@
   },
   "scripts": {
     "install": "composer install",
-    "build": "gulp",
+    "build": "gulp buildDev",
+    "dev": "gulp",
+    "gulp": "gulp",
```

Comment:
I added `gulp` so it can be used to run other tasks besides the default task. For example, `npm run gulp -- images` would run the image task in isolation. 

While `npm run dev -- images` would do the same thing as they both map to `gulp` I think `gulp` is more intuitive.

If we are going to remove one I would advocate for removing `dev` rather than `gulp` and update the documentation so that `npm run gulp` is the command we recommend for development.